w2w/w2w.py

```diff
@@ -1127,8 +1142,17 @@ def _adjust_greenfrac_landusef(
     # Adjust GREENFRAC and LANDUSEF
     # GREENFRAC is set as average / month from GREENFRAC
     # of original urban pixels
+    
+    #Define pixels that are urban in the original data
+    if orig_num_land_cat == 61:
+        urban_cat_list = [51, 52, 53, 54, 55, 56, 57, 58, 59, 60, urban_cat]
+    elif orig_num_land_cat == 41:
+        urban_cat_list = [31, 32, 33, 34, 35, 36, 37, 38, 39, 40, urban_cat]
+    else:
+        urban_cat_list = [urban_cat]
+    
```

Comment:
see previous comment. Perhaps here the urban_cat_list can be defined depending on Info.LCZ_BUILT, that by default has [1 .. 10], but might also be [1 .. 11]?